### ISSUE TITLE
When using download-status, files don't get extracted

### ISSUE THREAD

**Issue by piranna** [kevva/download#62]
By using

```Javascript
  var Download = require('download')
  var progress = require('download-status')

  const NODEJS_VERSION = "v0.11.14"
  const NODEJS_URL="http://nodejs.org/dist/"+NODEJS_VERSION+"/node-"+NODEJS_VERSION+".tar.gz"
  const SRC_DIR = 'deps/node'

  Download({ extract: true, strip: 1 })
  .get(NODEJS_URL, SRC_DIR)
//  .use(progress())
  .run(function(error)
  {
    ...
  }
```

the source code of Node.js it's downloaded and extracted correctly at ```deps/node```, but by using

```Javascript
  var Download = require('download')
  var progress = require('download-status')

  const NODEJS_VERSION = "v0.11.14"
  const NODEJS_URL="http://nodejs.org/dist/"+NODEJS_VERSION+"/node-"+NODEJS_VERSION+".tar.gz"
  const SRC_DIR = 'deps/node'

  Download({ extract: true, strip: 1 })
  .get(NODEJS_URL, SRC_DIR)
  .use(progress())
  .run(function(error)
  {
    ...
  }
```

the code don't get extracted and the .tar.gz only gets stored at ```deps/node``` instead.

**Comment by kevin-smets** [comment]
Ah damn, just encountered this one and I lost a good amount of time on it. Reverted to download v 3.3.0 and all is well (for now).

**Comment by floatdrop** [comment]
@piranna @kevva this is fixed in `4.1.4`.

**Comment by piranna** [comment]
I'll test it later.
